Okta.Xamarin/Okta.Xamarin/OktaXamarinUserAgent.cs

```diff
@@ -0,0 +1,16 @@
+﻿// <copyright file="OktaXamarinUserAgent.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Reflection;
+
+namespace Okta.Xamarin
+{
+	public class OktaXamarinUserAgent
+    {
+        private static Lazy<string> userAgent = new Lazy<string>(() => $"Okta-Xamarin-Sdk/{Assembly.GetExecutingAssembly().GetName().Version}");
```

Comment:
In the future, you might want to track if devs are using iOS or Android, so I'd consider that each project (iOS and Android) informs their specific user-agent/versions.